hammer_cli_foreman_tasks.gemspec

```diff
@@ -21,5 +21,5 @@ DESC
   s.require_paths = ["lib"]
 
   s.add_dependency "powerbar", ">= 1.0.11", "< 3.0"
-  s.add_dependency "hammer_cli_foreman", "> 0.1.1", "< 1.0.0"
+  s.add_dependency "hammer_cli_foreman", "> 0.1.1", "< 3.0.0"
```

Comment:
should we remove the limit so we don't hit this again when foreman 3.0 is released?

Reply:
Major versions can indicate breakage if semver is followed. I think it's safer to be more defensive because there's no guarantee this will actually work with 3.0.

Reply:
most hammer plugins don't require a specific version, but we can remove the limit in a separate pr if needed.